sound/soc/sof/ipc4-priv.h

```diff
@@ -69,6 +69,7 @@ struct sof_ipc4_fw_library {
  * @max_num_pipelines: max number of pipelines
  * @max_libs_count: Maximum number of libraries support by the FW including the
  *		    base firmware
+ * @max_kcps: maximum cycles per second * 1000, iow the maximum core frequency
```

Comment:
why we need to introduce max_cpc ?  The cpc should be set in manifest or we should return error to notify developer to fix this issue

Reply:
this is max_**kcps**, not cpc, this is not set in manifest, it is the maximum clock speed the DSP can run if we want to put it in human readable terms: **K**ilo **C**ycles **P**er **S**edond.
CPC is is cycles per chunk and a chunk is 1 sec (at least I have only seen this convention).